contracts/src/common/ObsERC20.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../messaging/IMessageBus.sol";
+
+// This is an implementation of a canonical ERC20 as used in the Obscuro network
+// where access to data has to be restricted.
+contract ObsERC20 is ERC20 {
+    address bridge = 0xdeB34A740ECa1eC42C8b8204CBEC0bA34FDD27f3;
```

Comment:
hardcoded address ? Will this be a problem ?

Reply:
This is a leftover from the bits that were used to test cross chain messages versus the old bridge. It will be removed with the PR that removes the old bridge.